CMakeLists.txt

```diff
@@ -48,6 +48,10 @@ if (NOT DEFINED WAMR_BUILD_TARGET)
   endif ()
 endif ()
 
+if (NOT COMPILING_WASM_RUNTIME_API)
+  set(COMPILING_WASM_RUNTIME_API 1)
+endif ()
```

Comment:
Seems no need to add these lines as COMPILING_WASM_RUNTIME_API is a macro used in the source files and `add_definitions(-DCOMPILING_WASM_RUNTIME_API=1)` is added later in this PR. Could you remove them?

Reply:
removed